.rubocop.yml

```diff
@@ -0,0 +1,9 @@
+---
+# Managed by modulesync - DO NOT EDIT
+# https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
+
+inherit_gem:
+  voxpupuli-test: rubocop.yml
+
+Layout/FirstHashElementIndentation:
```

Comment:
I don't object to this cop but I'm not a fan of this style.

Reply:
I don't have an opinion on the style—its main function in this patch is to keep the source readable after an autocorrect, because enforcing the current cop's settings produced some bizarre-looking extremes: https://github.com/voxpupuli/puppet-puppet_authorization/pull/1#discussion_r942942195.  If there's a better way to prevent rubocop from doing that, this setting can go away.

Reply:
It should format acceptably without the `{}`s around the `#with` method parameters. E.g.:
```ruby
.with(
  target: foo,
 content: bar,
)
```

Reply:
IIRC, [Ruby 3 requires the braces](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/) around positional hash arguments, to remove ambiguity with keyword arguments. The Ruby Style Guide mentions that [braces around an options hash are also required in 2.7](https://rubystyle.guide/#no-braces-opts-hash), although I think 2.7 just emits a warning.

Reply:
I think you can pass them as keyword arguments (with Ruby 3 they end up in `**kwargs` if the method accepts it). However, I'm a fan of:
```ruby
.with_target(foo)
.with_content(bar)
```

Reply:
Ruby3 does not allow named parameters to be mixed with an option hash. If the method doesn't have named parameters, there is no change in behavior. 

If this wasn't the case, a lot of voxpupuli spec code would be failing.

Reply:
I dislike .with_foo when testing more than one parameter on a resource as it is additional character to type and is visually repetitive.

Reply:
Updated; I went with @jhoblitt 's preference because it was simpler to substitute.